contract_manager/scripts/list_wormhole_contracts.ts

```diff
@@ -0,0 +1,40 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { DefaultStore, EvmWormholeContract } from "../src";
+
+const parser = yargs(hideBin(process.argv))
+  .usage("Usage: $0")
+  .options({
+    testnet: {
+      type: "boolean",
+      default: false,
+      desc: "Fetch testnet contracts instead of mainnet",
+    },
+  });
+
+async function main() {
+  const argv = await parser.argv;
+  const entries = [];
+  for (const contract of Object.values(DefaultStore.wormhole_contracts)) {
+    if (contract instanceof EvmWormholeContract) {
+      if (contract.getChain().isMainnet() === argv.testnet) continue;
```

Comment:
This is a minor nit but I definitely prefer:

```
if (contract instanceof EvmWormholeContract && contract.getChain().isMainnet() !== argv.testnet) {
```

on the line above -- the reason is that I generally try to avoid control-flow-breaking syntax like `continue`, because it makes it much harder to look through the code and understand that an early exit could have happened somewhere that isn't represented by the branch condition.